VTranscribe/Model/User.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  User.swift
+//  VTranscribe
+//
+//  Created by nguyen.minh.ducd on 5/4/19.
+//  Copyright © 2019 nguyen.minh.ducd. All rights reserved.
+//
+
+import ObjectMapper
+
+struct User {
+    var refreshToken = ""
```

Comment:
không gán luôn giá trị, tương tự cho accessToken

```suggestion
    var refreshToken: String
```